docs/CONFIGURATION.md

```diff
@@ -33,6 +33,10 @@ files in `/sys/module/hid_xpadneo/parameters`:
     * `16` if your controller boots in Linux mode (auto-detected, do not change manually)
     * `32` if you prefer to use Nintendo button mappings (i.e., 8BitDo controllers, defaults to off)
     * `64` if your controller has a awkwardly mapped Share button (auto-detected, do not set manually)
+* 'disable_chords' (default 0)
```

Comment:
While I get where "chord" comes from, I'm not sure if this is the correct word here - but I'm not a native speaker. Maybe it should be renamed to `disable_shift_mode`: It shifts the operation of the buttons to a different operation - just like the shift key on your keyboard.